lib/utils.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const fastCopy = require('fast-copy')
+const { createCopier } = require('fast-copy')
```

Comment:
How is this better than the `.default` version? We are still changing code to satisfy a dependency.

Reply:
fast-copy released a semver major with this change. we need the new version to unblock https://github.com/pinojs/pino-webpack-plugin/pull/54, so this is the only solution we have, which looks fairly reasonable to me